### PR TITLE
[9.x] Update to Symfony v6 and drop PHP 7.4 support

### PR DESCRIPTION
**Pull request by driesvints**

### CODE REVIEW

File: tests/Routing/RoutingUrlGeneratorTest.php [hunk 1]
```diff
@@ -508,7 +508,7 @@ public function testHttpsRoutesWithDomains()
 
     public function testRoutesWithDomainsThroughProxy()
     {
-        Request::setTrustedProxies(['10.0.0.1'], SymfonyRequest::HEADER_X_FORWARDED_ALL);
```
Comment (GrahamCampbell) [inline tests/Routing/RoutingUrlGeneratorTest.php:511]: this change we could make already on 8.x?

Comment (driesvints) [reply tests/Routing/RoutingUrlGeneratorTest.php:511]: Not sure

File: composer.json [hunk 2]
```diff
@@ -15,7 +15,7 @@
         }
     ],
     "require": {
-        "php": "^7.4|^8.0",
+        "php": "^8.0",
```
Comment (X-Coder264) [inline composer.json:18]: @driesvints Shouldn't this change be made in all Illuminate components `composer.json` too?

Comment (driesvints) [reply composer.json:18]: Yep seems I forgot to do that. Can you send in a PR? Otherwise I'll try to do that later this week.

Comment (X-Coder264) [reply composer.json:18]: Sure. I'll send one later today.

**Pull request merged**